astropy/coordinates/tests/test_erfa_astrom.py

```diff
@@ -85,8 +85,9 @@ def test_interpolation_nd():
 
         altaz = AltAz(location=fact, obstime=obstime)
```

Comment:
altaz no longer used?

Reply:
I think it is? Maybe this comment is outdated...

Reply:
I don't think it is: you do not use `altaz` any more in the `for` loop below...

Reply:
You mean the for loop that starts as below?

```python
for frame, tcode in zip([altaz, cirs, gcrs], ['apio', 'apco', 'apcs']):
```

because `altaz` is definitely used in that loop.

Reply:
Duh, blinded as I was by seeing `altaz` changed to `cirs`, I didn't notice the other `altaz` - sorry!